internal/graphapi/resolver_test.go

```diff
@@ -108,7 +108,7 @@ func TestNodeResolving(t *testing.T) {
 			TestName:  "invalid prefix",
 			query:     `{"query": "{ node(id: \"invalidtest-987\") { __typename id } }" }`,
```

Comment:
we should make the prefix a single letter since that will actually trigger the invalid prefix error we are trying to check for here

Reply:
Fixed!